app/assess/data.py

```diff
@@ -162,11 +162,11 @@ def get_bulk_accounts_dict(account_ids: List, fund_short_name: str):
             del debug_user_config["highest_role"]
```

Comment:
```suggestion
            debug_user_config["highest_role_map"] = {fund_short_name: fund_short_name + "_LEAD_ASSESSOR"}
            del debug_user_config["highest_role"]
```

Reply:
can we make DEBUG_USER_ROLE dynamic. suggestion to replace Line 161

Reply:
I see what you mean.  I think it would be `{"COF": "LEAD_ASSESSOR"}` as `highest_role` is used only for presentation now afaik.  So doesn't require the `COF_` or `NSTF_` prefix.